SwiftGen.xcodeproj/xcshareddata/xcschemes/swiftgen.xcscheme

```diff
@@ -40,8 +40,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "NO"
-      codeCoverageEnabled = "YES">
+      shouldUseLaunchSchemeArgsEnv = "NO">
```

Comment:
Needs rebase after the merge of #703, in order to re-include the code coverage in there and not re-remove it I guess?

Reply:
🤦‍♂️